pandas/io/pytables.py

```diff
@@ -1027,6 +1028,7 @@ def put(
             data_columns=data_columns,
             encoding=encoding,
             errors=errors,
+            track_times=track_times,
```

Comment:
same

Reply:
same = you mean docstring? I do not see any docstring here.